.docerignore

```diff
@@ -0,0 +1,12 @@
+.dockerignore
```

Comment:
Fix the typo in the filename ~`.docerignore`~ -> `.dockerignore`.

Reply:
addressed